### DISCUSSION TITLE
NoiseModelling 4.0.1

### DISCUSSION THREAD

**Discussion by nicolas-f** [Universite-Gustave-Eiffel/NoiseModelling#505] [General]
## Updates
- Fix discrete directivity of sound sources
- Add WPS script to export polar graph of sound source directivity
- Add tutorial "Point Source noise mapping"
- In the tutorial export 3D rays
- handle .osm .osm.gz file formats

## Packaging
- `NoiseModelling_4.0.1.zip` : cross-platform version, with GUI *(Graphic User Interface)*
- `NoiseModelling_4.0.1_install.exe` : windows installer, with GUI
- `NoiseModelling_4.0.1_without_gui.zip` : version without GUI. Useful to run NoiseModelling using command lines

<hr /><em>This discussion was created from the release <a href='https://github.com/Universite-Gustave-Eiffel/NoiseModelling/releases/tag/v4.0.1'> ## Packaging - `NoiseModelling_4.0.0.zip` : cross-platform version, with GUI *(Graphic User Interface)* - `NoiseModelling_4.0.0_install.exe` : windows installer, with GUI - `NoiseModelling_4.0.0_without_gui.zip` : version without GUI. Usefull to run NoiseModelling using command lines</a>.</em>

**Comment by qgoestch** [comment]
Hi, 
When running `startup_linux_mac.sh` on Ubuntu 20.04.5 LTS, I get the following error : 
```
The GEOSERVER_HOME environment variable is not defined
This environment variable is needed to run this program
Please set it to the directory where geoserver was installed
```
Do I have to install Geoserver?

**Comment by gpetit** [comment]
Hi @qgoestch,

Just to be sure, can you try to execute ``echo $JAVA_HOME`` in your Terminal.
In parallel, have you read the doc here : https://noisemodelling.readthedocs.io/en/latest/Requirements.html#linux-or-mac ?

**Reply by qgoestch** [reply]
I have read that already, and I get `/usr/lib/jvm/java-11-openjdk-amd64/` when I execute `echo $JAVA_HOME`

**Reply by gpetit** [reply]
Can you tell us if you launch NM thanks to command lines (``sh startup_linux_mac.sh``) directly from the ``bin/`` folder ?

**Reply by qgoestch** [reply]
From a directory I try to launch NM thanks to `NoiseModelling_4.0.1/bin/startup_linux_mac.sh`

**Reply by qgoestch** [reply]
If I go to the `bin/` folder and run the bash file itself, NM launches successfully

**Reply by gpetit** [reply]
Ok, I got the same error. 
Thanks for the warning. We will modify the doc to avoid this problem

**Reply by qgoestch** [reply]
Thanks for the answers, see you !
